### DISCUSSION TITLE
Where I can get the template https://ui.shadcn.com/examples/forms

### DISCUSSION THREAD

**Discussion by softshipper** [shadcn-ui/ui#5461] [General]
Hi all,
Could someone please tell me where can I get the https://ui.shadcn.com/examples/forms template? I couldn't find the option to show the code.
I am greatly appreciated for the help.
Thanks

**Comment by 4rpan** [comment]
https://github.com/shadcn-ui/ui/tree/main/apps/www/app/(app)/examples/forms
Follows the next app dir convention
